.github/workflows/test.yml

```diff
@@ -67,9 +67,8 @@ jobs:
       - name: Cache backend coverage results
         uses: actions/cache@v3
         with:
-          path: |
-            backend/build/**
-          key: backend-coverage-${{ hashFiles('backend/build/**') }}
+          path: backend/build/**
```

Comment:
nit but I noticed that the path got changed here, but all other places have the pipe. was that intentional?